pages/expenses.js

```diff
@@ -84,11 +95,23 @@ class ExpensePage extends React.Component {
 
   buildFilterLinkParams(params) {
     return {
-      ...pick(this.props, ['collectiveSlug', 'limit', 'parentCollectiveSlug', 'tag', 'type']),
+      ...pick(this.props, ['collectiveSlug', 'limit', 'parentCollectiveSlug', 'tag', 'type', 'searchTerm']),
       ...params,
     };
   }
 
+  handleSearch(event) {
+    const searchInput = event.target.elements.q;
```

Comment:
Can't we directly get the value through `event.target.value`?

Reply:
@Betree : No unfortunately; `event.target` is a `<form>` element and `event.target.value` will be undefined.

Reply:
I see, thanks for the explanation. That's not a great API from `SearchForm` but it works